models/staging/products/stg_products.sql

```diff
@@ -0,0 +1,14 @@
+with src as (
+  select * from  {{ source ('coffee_shop', 'products') }}
+),
+
+transformed as (
+  select
+    id as product_id,
+    name as product_name,
+    category as product_category,
+    created_at
```

Comment:
Maybe my own personal preference but I strongly dislike leaving generic names like "created_at" I renamed this to "price_start_date" but I also combined products and prices into one. The date itself in the product raw table was useless because it was all the same.